config.example.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+return [
+    'iiif' => 'https://iiif.elifesciences.org/medium:',
```

Comment:
needs to be added to https://github.com/elifesciences/builder-private/blob/master/pillar/iiif.sls#L16